### ISSUE TITLE
Rule F4 Misses dimension_groups without descriptions

### ISSUE THREAD

**Issue by will-regal-voice** [looker-open-source/look-at-me-sideways#119]
The following example will pass Rule F4 despite not having a description
```
  dimension_group: start {
    type: time
    sql: ${TABLE}."START_DATE" ;;
    timeframes: [date, week, month, year, day_of_week, day_of_month]
  }
  ```

**Comment by fabio-looker** [comment]
Hi @will-regal-voice 

Because most users have LAMS set up in a CI environment, I'd consider introducing error conditions a breaking change.

I am currently considering two options for releasing this change:

1. Release it as a 2.x release, but require a flag in the manifest to opt-in to the newer logic
2. Release it as a 3.x pre-release, which would not require a manifest flag, but could require you to pin your install script to specific version until the actual 3.x release (hopefully within a month or two)

Any preference or alternative sugestion?

**Comment by will-regal-voice** [comment]
I think either one would be a great solution

**Comment by fabio-looker** [comment]
Should now be fixed in v2.1.6

See this example for how to opt-in to this change via the manifest file until v3:

https://github.com/looker-open-source/look-at-me-sideways/blob/master/__tests__/dummy-projects/19-f4-dimension-groups/opt-in/manifest.lkml
